README.md

```diff
@@ -1,77 +1,26 @@
-# Exercise 1
+# Readme
 
-## Intro
-During this bootcamp you'll be working on an evolving project, expanding it as you learn new concepts. It'll be a basic messaging system. Because the logistics of actually _delivering_ the messages can get rather complex, we'll leave that part out and just use an external service instead, `messageapp`, that will handle all that complexity for you.
 
-In this first exercise, you'll build a very simple API that'll interact with that external service. The service is available here as an external docker image.
+## Contract 
 
-###  1 - Clone the repository and spin up messageapp
-Along the instructions for the exercise, you'll find also a `docker-compose.yml` file. This file instructs docker-compose on how to build the distributed system this exercise requires. Take a closer look and you'll see there are a few lines commented out, those that start with `#`. You'll need to uncomment those lines later on. But for now, let's just spin up the `messageapp` service on its own.
+### Express endpoint table
 
-To do that, move to this folder on your command line, and execute this command:
-```sh
-$ docker-compose up
-```
-If you use linux, that should work out of the box, but remember that if you're using mac, you need to start the docker daemon first, or else you'll get an error on the console.
+- /messages 
+  - Method POST => 
+  - Response: 200 
+  - Error responses:
 
-The output of the command will show it downloads the image and spins up the container with the service. Once the service has started, you can check that it works correctly by sending a request. You can use whatever tool you are comfortable with, such as Postman or cURL, for instance. Try this request:
+     - 427 "Need destination & message keys"
```

Comment:
Why did you choose this status code?
let's say that you are the consumer of this error.
What will happen when you search http 427 status code ?

Reply:
Sorry I did't know, that there are predefined codes... 
I will search for the correct ones!!